### DISCUSSION TITLE
Maxspeed Tag Not Included

### DISCUSSION THREAD

**Discussion by LawrenceWlt** [osm-search/Nominatim#3360] [Q&A]
Am doing an nominatim reverse only import and wanted to have maxspeed in the extra_tags section. I edited the import-street.lua and added the maxspeed as follows: `extra_keys = {'wikipedia', 'wikipedia:*', 'wikidata', 'capital', 'area', 'maxspeed'}` in the flex.set_prefilters. After import, the tag is not available. Anything am doing wrong? Any help is appreciated.

Here is the result:
```
http://192.168.1.167:8080/reverse.php?lat=43.738649&lon=7.412378&zoom=16&extratags=1
<reversegeocode timestamp="Thu, 07 Mar 24 13:36:30 +0000" attribution="Data © OpenStreetMap contributors, ODbL 1.0. http://www.openstreetmap.org/copyright" querystring="lat=43.738649&lon=7.412378&zoomlevel=16&extratags=1">
<result place_id="100142" osm_type="way" osm_id="67923339" ref="Rue Honoré Labande" lat="43.7362968" lon="7.4150001" boundingbox="43.7351033,43.7362968,7.4150001,7.4154553" place_rank="26" address_rank="26">Rue Honoré Labande, Les Moneghetti, Monaco, 98000, Monaco</result>
<addressparts>
<road>Rue Honoré Labande</road>
<suburb>Les Moneghetti</suburb>
<city>Monaco</city>
<postcode>98000</postcode>
<country>Monaco</country>
<country_code>mc</country_code>
</addressparts>
<extratags/>
</reversegeocode>
```

**Comment by mtmail** [comment]
The returned street doesn't have any `maxspeed` set in OSM data https://www.openstreetmap.org/way/67923339 You might need to look at nearby streets, here's a map of maxspeed values in Monaco https://overpass-turbo.eu/s/1IjB

**Comment by LawrenceWlt** [comment]
Found out that I was using the wrong parameter. zoomlevel instead of zoom. The maxspeed is working as expected.
